src/strings/PowerFxResources.en-US.resx

```diff
@@ -4771,6 +4771,22 @@
     <value>The value passed to the '{0}' function cannot be changed.</value>
     <comment>Error message shown to the user if they try to use a mutation function (such as Patch / Collect) with a value that cannot be mutated.</comment>
   </data>
+  <data name="ErrorResource_ErrViolatedSimpleConstraintAccess_ShortMessage" xml:space="preserve">
+    <value>{0} cannot be accessed in expressions on this property.</value>
+    <comment>Error Message. This property restricts the set of values that can be referenced to just  </comment>
+  </data>
+  <data name="ErrorResource_ErrViolatedSimpleConstraintAccess_LongMessage" xml:space="preserve">
+    <value>This property restricts access to values outside of ThisItem to ensure good performance. {0} is not a value that can be accessed here.</value>
```

Comment:
>ThisItem [](http://example.com/codeflow?start=63&length=8)

Shouldn't we allow references to ThisRecord as well? I understand the intent behind this error message, but it should be reworded.